packages/components/src/screen-reader-text/README.md

```diff
@@ -0,0 +1,9 @@
+# ScreenReaderText
+
+A component used to render text intended for a screen reader, not visible elsewhere.
```

Comment:
You can reference WordPress docs here, maybe also copy some parts of that document:
https://make.wordpress.org/accessibility/handbook/markup/the-css-class-screen-reader-text/